docs/targeted-tutorials/read-local-data-in-k8s-silo.md

```diff
@@ -0,0 +1,78 @@
+> - It is probably not a good idea to expose **all** of the local file system.
+
+Now we are going to create a [Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) (PV) and the associated Claim (PVC), and deploy it.
+
+- To create the PV, first make sure that the `path` value of the `hostPath` entry in [pv.yaml](../../mlops/k8s_templates/pv.yaml) matches the `containerPath` value of the `extraMounts` in [k8s_config.yaml](../../mlops/k8s_templates/k8s_config.yaml). Then run: `kubectl apply -f ./mlops/k8s_templates/pv.yaml`.
+- To create the PVC, first make sure that the `name` in the `metadata` of [pvc.yaml](../../mlops/k8s_templates/pvc.yaml) matches the `name` of the `claimRef` in [pv.yaml](../../mlops/k8s_templates/pv.yaml). Also make sure that the metadata has the following `labels` and `annotations`, as **these are the key parts that are [required](https://github.com/Azure/AML-Kubernetes/blob/master/docs/pvc.md) for the Azure ML job to be able to access the local data**:
+    ```yaml
+    labels:
+        app: demolocaldata
+        ml.azure.com/pvc: "true"
+    annotations:
+        ml.azure.com/mountpath: "/mnt/localdata" # The path from which the local data will be accessed during the Azure ML job. You can change that to a different path if you want.
+    ```
+    Then run: `kubectl apply -f ./mlops/k8s_templates/pvc.yaml`.
+- Finally, to deploy, first make sure that the `claimName` in [deploy_pvc.yaml](../../mlops/k8s_templates/deploy_pvc.yaml) matches the PVC name in [pvc.yaml](../../mlops/k8s_templates/pvc.yaml), and that the `mountPath` matches the `path` in [pv.yaml](../../mlops/k8s_templates/pv.yaml). Then run: `kubectl apply -f ./mlops/k8s_templates/deploy_pvc.yaml`.
+
+> **:checkered_flag: Checkpoint**:\
+> Before moving forward, we recommend you check that the local data can indeed by accessed from the k8s cluster. To do so, start by getting the name of the pod that was created by the deployment by running `kubectl get pods`. Then, open a bash session on the k8s by running `kubectl exec -it <your-pod-name> bash`. Finally, run `ls <path-in-docker>` to check that the data in that folder are indeed visible (if you didn't change the default values in the yaml files mentioned above, then your `<path-on-docker>` should be `/localdata` - it is simply the `path` in [pv.yaml](../../mlops/k8s_templates/pv.yaml)).
+
+
+### 2. Create an external silo based on your k8s cluster
+At this point, you should have a properly configured k8s cluster. You now need to create an **external silo** (on-premises) based on your k8s. Do so by following [these instructions](../provisioning/external-silos.md). Make note of the Azure ML compute and datastore names for your newly created silo.
+
+> **:checkered_flag: Checkpoint**:\
+> Before moving forward, we recommend you make sure your silo is functional by successfully running the test job mentioned in the instructions for this step.
+
+
+### 3. Configure and run a test job
```

Comment:
could we leave this part out of the PR? as well as remove the examples/components/ and examples/pipelines/ ?

I'd love to merge the changes without that, for now.

Instead of a "read local data" pipeline, I'd suggest creating a "on_prem_hello_world" which would demo how to create local data in the context of a full training pipeline

Reply:
all right, removed the small pipeline and component. Let's talk on Monday and clarify the scope of the "on_prem_hello_world" for a subsequent PR